geom/tests/Test_GridGet.pf

```diff
@@ -0,0 +1,46 @@
+#include "MAPL_TestErr.h"
```

Comment:
This seems like an odd test for this PR.  Was it just forgotten in earlier PR?  (I.e., missed the git add step?)

Reply:
Yes, it got left out from #3589